### PR TITLE
Masonry layout

### PR DESCRIPTION
**Pull request by Jessmaxim303**
Hey everyone, I was able to get the masonry layout back up but wasn't able to do it packery. I can get packery to work on other projects so I have a pretty good understanding of how it works. I'm thinking it has something to do with webpack. I only added properties to .grid and .grid-item.

### CODE REVIEW

File: src/css/styles.css [hunk 1]
```diff
@@ -62,17 +66,33 @@ canvas {
...
+.grid:after {
+  content: '';
+  display: block;
+  clear: both;
+}
+
+.grid-item {
+  float: left;
+  width: 100%;
+  break-inside: avoid;
```
Comment (Dustin-Mikusko) [inline src/css/styles.css]: Curious what this does. I commented it out and it makes the bottom of each section appear above the tile. Interesting!

**Comment by Dustin-Mikusko** [comment]
Nice work Jesse! I'll move forward a little tonight working on media queries for smaller screens

**Comment by Dustin-Mikusko** [comment]
This PR closes #18 and closes #1

File: src/scripts.js [hunk 2]
```diff
@@ -65,6 +65,7 @@ function dropYear(dates) {
   return reformattedDates
 }
 $(document).ready(function () {
+  console.log('ready!')
```
Comment (adamsjr8576) [inline src/scripts.js]: I think we should try and keep our main clear of console.logs whenever we are pushing up to it.

File: src/css/styles.css (hunk 1 above)
Comment (adamsjr8576) [reply src/css/styles.css]: I am not totally following what's going on here either. I think we should still try and figure out how to relink the packery package once we get more familiar with how webpack works.

**Pull request merged**